_translator-files/po/de/Installation-for-Linux.po

```diff
@@ -57,7 +58,7 @@ msgstr "Stelle sicher, dass du die [Erste Schritte](Getting-Started) Seite berei
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:15
 msgid "Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first."
-msgstr "Updatest du? Es macht Sinn die [Konfiguration](Software-Manual#backing-up-jamulus) zu sichern."
+msgstr "Updatest du? Es macht Sinn die [Konfiguration](Software-Manual#Sicherung von Jamulus) zu sichern."
```

Comment:
For these links to specific paragraphs to work, apart from coinciding word for word, the words must be linked by hyphens (see the source phrase) and no capital letters.

Reply:
Noted and corrected in all files
how to handle ","
Example: You all sound OK, but it's difficult to keep together"

Reply:
Yes  - as I say you can look at other translations or just the source text to see what it should look like.